pkg/kapp/cmd/app/list.go

```diff
@@ -102,9 +104,11 @@ func (o *ListOptions) Run() error {
 			return err
 		}
 
+		appName := strings.TrimSuffix(item.Name(), app.AppSuffix)
```

Comment:
Should we move this inline?

Reply:
can do! Wanted to leave it like this so you could very clearly see what has to happen with this way of implementing this change.

Reply:
Or should we do this in the List function itself?

Reply:
Good call! I actually didn't see the List() call earlier, (still exploring the code). I will move it there, it makes more sense there